contents/data_vis_aog.md

```diff
@@ -0,0 +1,238 @@
+# Data Visualization with AlgebraOfGraphics.jl {#sec:aog}
+
+> **_NOTE:_**
+> Don't forget to check [`AlgebraOfGraphics.jl` documentation](https://aog.makie.org).
```

Comment:
This is a bit unclear. Our books is a more structured introduction to AoG, right? So why point people to the docs already at the start of our chapter?

Reply:
Maybe we should move this to the end of the chapter?

Reply:
Yes good idea